regexes/client/mediaplayers.yml

```diff
@@ -65,6 +65,10 @@
   name: 'SubStream'
   version: '$1'
 
+- regex: 'Sonos/([\d\.-]+)?'
+  name: 'SONOS'
```

Comment:
add a test into this PR
inset test in file `device-detector/Tests/fixtures/mediaplayer.yml`

is this the detect of a Sanos client? (sonos.com)

Reply:
Yes this detection is for SONOS Players